src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2920,6 +2968,21 @@ def _check_cluster_autoscaler_flag(enable_cluster_autoscaler,
             raise CLIError('min-count and max-count are required for --enable-cluster-autoscaler, please use the flag')
 
 
+def _validate_autoscaler_update_counts(min_count, max_count, node_count, is_enable_or_update):
+    """
+    Validates the min, max, and node count when performing an update
+    """
+    if min_count is None or max_count is None:
+        if is_enable_or_update:
+            raise CLIError('Please specify both min-count and max-count when --enable-cluster-autoscaler or '
+                           '--update-cluster-autoscaler is set.')
+    if min_count is not None and max_count is not None:
+        if int(min_count) > int(max_count):
+            raise CLIError('value of min-count should be less than or equal to value of max-count.')
+        if int(node_count) < int(min_count) or int(node_count) > int(max_count):
+            raise CLIError("current node count '{}' is not in the range of min-count and max-count.".format(node_count))
```

Comment:
Minor problem: Sometimes you capitalize the first letter, sometimes you use lowercase.

Reply:
I made it consistent. I left it lower case when the sentence starts with the flag name.